src/components/PopupWithForm.js

```diff
@@ -19,9 +22,20 @@ export default class PopupWithForm extends Popup {
     return inputObj;
   }
 
+  renderLoading(isLoading, loadingText = "Saving...") {
+    console.log(loadingText + " is the value of isLoading");
+    if (isLoading) {
+      this._submitButtonText.textContent = loadingText;
+    } else {
+      this._submitButtonText.textContent = this._submitButtonText;
```

Comment:
This code is incorrect because `this._submitButtonText` is a button but you need to set here the initial button **text**